src/sleep/sleep.c

```diff
@@ -263,40 +263,44 @@ static int execute(
 }
 
 static int execute_s2h(const SleepConfig *sleep_config) {
-        _cleanup_close_ int tfd = -1;
-        struct itimerspec ts = {};
+        _cleanup_close_ int betfd = -1;
```

Comment:
these two should move inside the loop - also any specific reason to rename them?

Reply:
ahh. yes. they should be in loop. will update. reason to rename : added "be" as battery estimate for my understanding when the other timer was also there. Could name it back to the previous one. No issue.